build_windows.md

```diff
@@ -1,65 +1,97 @@
-a [Windows 11 development machine](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/) or a
-configuration close to this one. The Podman Windows client installer bundles several tools, which are unnecessary for Podman builds, but this
-set of packages is well aligned with GitHub's `windows-latest` offerings. Some of the tools will still be missing from
-this distribution and will have to be manually added after this installation completes.
+```pwsh
+$env:Path += ";C:\Program Files\Go\bin\;C:\Program Files\Git\cmd\"
+```
 
-## Install Pandoc
+### Pandoc
 
+[Pandoc](https://pandoc.org/) is used to generate Podman documentation. It is required for a full build of the Podman Windows installer but can be ignored to build the Podman Windows client or a minimal build of the Podman Windows Installer.
 Pandoc could be installed from https://pandoc.org/installing.html When performing the Pandoc installation one, has to choose the option
 "Install for all users" (to put the binaries into "Program Files" directory).
 
-## Install WiX Toolset v3 (is preinstalled in GitHub runner)
-The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
+### WiX Toolset v3
+
+[WiX Toolset](https://wixtoolset.org) **v3** is used to develop and build Podman Windows installer. It's not required to build the Podman Windows client.
+Version 3 of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
 an additional installation of .NET Framework 3.5 in advance
 ([instructions for adding .NET Framework 3.5 via enabling the Windows feature](https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel))
 
-## Building and running
+### Virtualization Provider
 
-### Install git and go
+Running Podman on Windows requires a virtualization provider. The supported providers are the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/) and [Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v). At least one of those two is required to test podman on a local Windows machine.
 
-To build Podman, the [git](https://gitforwindows.org/) and [go](https://go.dev) tools are required. In case they are not yet installed,
-open a Windows Terminal and run the following command (it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed):
+#### WSL
 
-```
-winget install -e GoLang.Go Git.Git
-```
-
-:information_source: A terminal restart is advised for the `PATH` to be reloaded. This can also be manually changed by configuring the `PATH`:
+WSL can be installed on Windows 10 and Windows 11, including Windows Home with the following command, from a PowerShell or Windows Command Prompt terminal in *administrator mode*:
```

Comment:
Again, I think we should only mention PowerShell since that's what we've tested on, and also because our scripts are in Powershell.

Reply:
PowerShell is pre-installed in Windows 10/11. There is no extra package to install so I would not mention that as a "requirement". What do you think?

Reply:
I have mentioned "PowerShell terminal" in the OS requirements.